deployments/deploy.js

```diff
@@ -4,11 +4,12 @@ async function main() {
     const UsersInfo = await ethers.getContractFactory("UsersInfo");
     const user_info = await UsersInfo.deploy();
```

Comment:
Consider removing the unnecessary console log statement, as it does not add any value and clutters the output.